dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsJdbcTask.java

```diff
@@ -196,6 +208,19 @@ private String getSql(@Nonnull JdbcHandle handle) {
                 + "AND L.StartTime < CAST('%s' AS TIMESTAMP)\n",
             SQL_FORMAT.format(interval.getStart()), SQL_FORMAT.format(interval.getEndExclusive())));
 
+    logDateColumn.ifPresent(
+        column -> {
+          String date = "CAST('" + SQL_DATE_FORMAT.format(interval.getStart()) + "' AS DATE)";
```

Comment:
Do we know that LogDate == cast(StartTime as DATE) or do we risk losing some statements around midnight?

Reply:
No. I just suspect that the timezone would be the same for both and there will be no lost statements.